e2e/tests/migration_success.feature

```diff
@@ -0,0 +1,98 @@
+@manual
```

Comment:
Note: I did not review these in detail, but really looks like you've put a lot of thought into so ty!

Reply:
We'll double check everything closer to MainNet